Makefile.am

```diff
@@ -36,30 +38,31 @@ rdmaincludedir = $(includedir)/rdma
 rdmainclude_HEADERS =
 
 # internal utility functions shared by in-tree providers:
-common_srcs = \
-	src/common.c \
-	src/enosys.c \
-	src/rbtree.c \
-	src/fasthash.c \
-	src/indexer.c \
-	src/iov.c \
-	prov/util/src/util_atomic.c \
-	prov/util/src/util_attr.c   \
-	prov/util/src/util_av.c     \
-	prov/util/src/util_cq.c     \
-	prov/util/src/util_cntr.c   \
-	prov/util/src/util_domain.c \
-	prov/util/src/util_ep.c     \
-	prov/util/src/util_pep.c     \
-	prov/util/src/util_eq.c     \
-	prov/util/src/util_fabric.c \
-	prov/util/src/util_main.c   \
-	prov/util/src/util_poll.c   \
-	prov/util/src/util_wait.c   \
-	prov/util/src/util_buf.c    \
-	prov/util/src/util_mr.c     \
-	prov/util/src/util_ns.c     \
-	prov/util/src/util_shm.c
+common_srcs =				\
+	src/common.c			\
+	src/enosys.c			\
+	src/rbtree.c			\
+	src/fasthash.c			\
+	src/indexer.c			\
+	src/iov.c			\
+	prov/util/src/util_atomic.c	\
+	prov/util/src/util_attr.c	\
+	prov/util/src/util_av.c		\
+	prov/util/src/util_cq.c		\
+	prov/util/src/util_cntr.c	\
+	prov/util/src/util_domain.c	\
+	prov/util/src/util_ep.c		\
+	prov/util/src/util_pep.c	\
+	prov/util/src/util_eq.c		\
+	prov/util/src/util_fabric.c	\
+	prov/util/src/util_main.c	\
+	prov/util/src/util_poll.c	\
+	prov/util/src/util_wait.c	\
+	prov/util/src/util_buf.c	\
+	prov/util/src/util_mr.c		\
+	prov/util/src/util_ns.c		\
+	prov/util/src/util_shm.c	\
+	prov/util/src/util_mr_monitor.c
```

Comment:
Let's rename this file to either `util_monitor.c` or `util_mem_monitor.c`.
I prefer the second one. Seems this name reflects the purpose of this functionality in more detail

Reply:
I agree with that. The header file should probably be renamed as well.